src/musl-prepo.json

```diff
@@ -95,6 +96,39 @@
            // 	jmp *56(%rdi)           /* goto saved address without altering rsp */
             "data":"McCD/gER8EiLH0iLbwhMi2cQTItvGEyLdyBMi38oSItnMP9nOA=="
           }
+        },
+        "424c81f9f233a916a0786db40b288b5f":{
+          "text":{
+           // .text
+           // .global __clone
+           // .hidden __clone
+           // .type   __clone,@function
+           // __clone:
```

Comment:
I would remove these lines. They all relate to the definition of the symbol in ELF which isn't part of the repo fragment.

Reply:
Thanks, fixed in the commit https://github.com/SNSystems/musl-prepo/pull/10/commits/40103e52dfb82db7f19d3211ee05034e2d103809.